### ISSUE TITLE
Blocks access to published host ports

### ISSUE THREAD

**Issue by smessmer** [AkihiroSuda/cni-isolation#4]
Thanks for building this plugin, it's great for isolating containers from each other for security. I noticed one issue when using it: It doesn't only block access to the internal networks of other containers, it also blocks access from containers to public host ports (tried to access via the public host ip) if those are published from a different container. I would like my containers to be isolated from each other's non-published ports, but I want them to be able to talk to the ports that are open and published to the internet anyways. I'm not sure if this is due to the initial SYN packet being blocked by iptables or whether the ACK gets blocked.

Example:
---
Host: server.example.com

Container 1: publishes port 80 to host port 80. Connected to other containers via internal networks.

Container 2: Connected to other containers via internal networks. Cannot access server.example.com:80 even though that's public and other hosts can access it through the public ip.

**Comment by AkihiroSuda** [comment]
This plugin was merged into the upstream `firewall` plugin: https://github.com/containernetworking/plugins/commit/22dd6c553dbb9fd3e34502c0de16dd17c7461a83

I'm going to archive https://github.com/AkihiroSuda/cni-isolation soon, could you open an issue in https://github.com/containernetworking/plugins/ ?
